book/src/configuration.md

```diff
@@ -78,6 +78,7 @@ The following elements can be configured:
 | `diagnostics` | The number of warnings and/or errors |
 | `selections` | The number of active selections |
 | `position` | The cursor position |
+| `position-pct` | The cursor position as a percentage of the total number of lines |
```

Comment:
`position-percentage` is more readable and saving characters doesn't do us any favors any way.

Reply:
renamed per suggestion. Thanks!